src/make.ts

```diff
@@ -215,6 +259,15 @@ export abstract class Make // static class
             func(i);  
     }
     
+    /**
+     * 
+     * Iterates through a loop specified by the `numberOfTimes` while invoking the asynchronous callback function `func` with the specific index of the loop. 
```

Comment:
Executes an async function given numberOfTimes 
degreesOfParallelism = How many are functions are going to execute at a time.
1 = 1 at a time. 
default = all executed at the same time